tests/installation/await_install.pm

```diff
@@ -147,6 +150,9 @@ sub run {
             }
             next;
         }
+        if (match_has_tag("linuxrc_error")) {
+            die 'Installation cant continue. Check media or hardaware.';
```

Comment:
s/hardaware/hardware/

I think if any media are involved it's only the initial installation medium, so better make that "medium" instead of media

Reply:
Done